qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -36,6 +36,14 @@ def str_to_bool(val):
 class TestCephFSShell(CephFSTestCase):
     CLIENTS_REQUIRED = 1
 
+    def setUp(self):
+        super(TestCephFSShell, self).setUp()
+        self.fs.mon_manager.raw_cluster_cmd('config', 'set', 'client',
```

Comment:
shouldn't this be `... 'set', 'cephfs-shell'`?

Reply:
Also, https://tracker.ceph.com/issues/43247 revealed setting debug on the shell persisted to affect other tests. But I'm thinking for these tests we want to have debugging on for everything. Why not add that here too.

Reply:
> shouldn't this be ... 'set', 'cephfs-shell'?

I was expecting the same when I added this [block of code](https://github.com/ceph/ceph/pull/29964/files?file-filters%5B%5D=.cc&file-filters%5B%5D=.sh#diff-3d7a065928b2852c228ffe669d7633bbR8284), but when I tested my patch `ceph config set client` worked and `ceph config set cephfs-shell` didn't. I couldn't figure out why that was the case, but when I did not get review on it from you I thought only `ceph config set client` was, somehow, supposed to be work.

Reply:
> Also, https://tracker.ceph.com/issues/43247 revealed setting debug on the shell persisted to affect other tests.

I was working on this yesterday, debug option had nothing to do with TestSnapshots's failure.

> But I'm thinking for these tests we want to have debugging on for everything. Why not add that here too.

I agree. I'll push that change.